src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/hf/HuggingFaceRequestBody.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.research.testspark.tools.llm.generation.hf
+
+import org.jetbrains.research.testspark.core.data.ChatMessage
+
+data class Parameters(
+    val topProbability: Double,
+    val temperature: Double,
+)
+
+data class HuggingFaceRequestBody(
+    val messages: List<ChatMessage>,
+    val parameters: Parameters,
+)
+
+fun HuggingFaceRequestBody.toMap(): Map<String, Any> {
+    return mapOf(
+        "inputs" to this.messages.joinToString(separator = "\n") { it.content },
```

Comment:
Could you add a small Javadoc comment for this method and a link to the platform docs where the `parameters` are described?

Reply:
Addressed in https://github.com/JetBrains-Research/TestSpark/pull/212/commits/e93a3ed998c961d36c5675f1cff1e41b633f74a2